lib/puppet/file_serving/terminus_helper.rb

```diff
@@ -29,4 +29,16 @@ def path2instances(request, *paths)
       path2instance(request, base_path, :ignore_source_permissions => true, :relative_path => file)
     end
   end
+
+  def self.unescape_url(url)
+    url.sub(/^url=/,'')
+  end
+
+  def self.escape_url(url)
+    if url =~ /^https?:/
+      "url=#{url}"
+    else
+      url
+    end
```

Comment:
Hmm, the mutation looks kind of neat. But that's just me :)

So if the indicated line is broken, does that mean that this code path is not in active use by anything anymore? I kind of feel like just replacing it with a "don't touch HTTP URLs" behavior.